djstripe/migrations/0001_initial.py

```diff
@@ -1603,15 +1603,15 @@ class Migration(migrations.Migration):
                         help_text="The latest charge generated for this invoice, if any.",
                         null=True,
                         on_delete=django.db.models.deletion.CASCADE,
-                        related_name="latest_invoice",
+                        related_name="latest_%(class)s",
```

Comment:
What is the first Django version to support this style of related name?

Reply:
`3.2.0`. Also none of the tests failed.